lib/httplib/httpheaders.go

```diff
@@ -106,11 +106,11 @@ func combineCSPMaps(cspMaps ...cspMap) cspMap {
 	return combinedMap
 }
 
-// getContentSecurityPolicyString combines multiple CSP maps into a single
+// GetContentSecurityPolicyString combines multiple CSP maps into a single
 // CSP string, alphabetically sorted by the directive key.
 // When multiple of the input cspMaps have the same key, their
 // respective lists are concatenated.
-func getContentSecurityPolicyString(cspMaps ...cspMap) string {
+func GetContentSecurityPolicyString(cspMaps ...cspMap) string {
```

Comment:
If you're going to export this you should also export the `cspMap` type or come up with a new signature. An exported function that requires an unexported type is difficult to use.

Reply:
https://github.com/gravitational/teleport/pull/44529/commits/691b07069dedae8d1c376a604a1df1b0f78dc63b